backend/application/syson-sysml-export/src/test/java/org/eclipse/syson/sysml/export/SysMLModelToTextSwitchTest.java

```diff
@@ -671,6 +673,20 @@ public void attributeUsage() {
         this.assertTextualFormEquals("attribute attribute1;", attributeUsage);
     }
 
+    @Test
+    public void attributeUsageWithBinaryOperatorExpression() {
+        AttributeUsage attributeUsage = this.builder.createWithName(AttributeUsage.class, ATTRIBUTE1);
```

Comment:
Why this part does not belong to the TestModel?

Reply:
done